src/app/pages/NFTByFtxId/HeroleteItem/HeroleteItem.tsx

```diff
@@ -25,22 +26,49 @@ export const HeroleteItem = () => {
       <div className={styles.heroleteItem}>
         <div>
           <RibbonIcon fill={colors.white} />
-          <Typography component="div" variant="subtitle1">
-            Tier
+          <Typography component="div" variant="subtitle1" className={styles.heroleteItem__category}>
+            Tier:
           </Typography>
           <Typography component="div" variant="subtitle1" className={styles.heroleteItem__info}>
             {nft?.attributes.Tier}
           </Typography>
         </div>
         <div>
           <BackgroundIcon fill={colors.white} />
-          <Typography component="div" variant="subtitle1">
-            Background
+          <Typography component="div" variant="subtitle1" className={styles.heroleteItem__category}>
+            Background:
           </Typography>
           <Typography component="div" variant="subtitle1" className={styles.heroleteItem__info}>
             {nft?.attributes.Background}
           </Typography>
         </div>
+        <div>
```

Comment:
as we are repeating this structure, what do you think about creating an array with items to render and then use a map ?

Reply:
I think it doesn't worth it for three elements. It's a very statical content and we're saving (a very little) computational work